sysl2/sysl/grammar/SyslParser.g4

```diff
@@ -384,8 +384,10 @@ view
     (ARROW_RIGHT view_return_type)?
     ( attribs_or_modifiers? COLON expr_block | abstract_view {$abstractView=true;} );
 
-alias: ALIAS name_str attribs_or_modifiers? COLON
-        INDENT annotation* (types | collection_type) DEDENT;
+alias: ALIAS name_str attribs_or_modifiers? COLON (
+        (annotation* (types | collection_type))
+        | (INDENT annotation* (types | collection_type) DEDENT)
```

Comment:
these two lines should be combined into another rule to remove duplication:
```
alias_def: annotation* (types | collection_type)
alias: ALIAS name_str attribs_or_modifiers? COLON (alias_def | INDENT alias_def DEDENT)
```

Otherwise this PR looks fine

Reply:
need a way to access `ctx.alias_def.annotation()` in the listener in order to achieve this